### PR TITLE
chore: remove build files

### PR DESCRIPTION
**Pull request by ricardogoncalves89**

### CODE REVIEW

**Review by djcouto — APPROVED** [review]
🚀

**Review by pedrorebelocosta — APPROVED** [review]

File: packages/angular-ui-components/src/kv-ui-components.module.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { defineCustomElements } from '@kelvininc/ui-components/loader';
-import { KvActionButton, KvSwitchButton, KvLink } from './generated/component';
+import { KvActionButton, KvSwitchButton } from './stencil-generated/component';
```
Comment (miguelpinto98) [inline packages/angular-ui-components/src/kv-ui-components.module.ts:4]: KvLink should be removed?

Comment (ricardogoncalves89) [reply packages/angular-ui-components/src/kv-ui-components.module.ts:4]: nop. THX

Comment (ricardogoncalves89) [reply packages/angular-ui-components/src/kv-ui-components.module.ts]: fixed ;)

**Review by miguelpinto98 — APPROVED** [review]
🚀

**Review by pedrorebelocosta — APPROVED** [review]

**Comment by pedrorebelocosta** [comment]
🚀

**Pull request merged**